components/ImpactBanner.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Card, CardContent } from "@/components/ui/card-impact";
+
+export const ImpactBanner = (): JSX.Element => {
```

Comment:
maybe we rename this on the figma&here to be more general like KPI Banner? But let this live for now